### DISCUSSION TITLE
秒传的时候电脑断电关闭

### DISCUSSION THREAD

**Discussion by gujin162** [AlistGo/alist#6462] [Q&A]
网盘A向网盘B秒传的时候，电脑突然断电，重启电脑后，再次开启aalist进行秒传，网盘B中已经秒传过来的文件是不是重新从网盘A复制？

**Comment by pongfcnkl** [comment]
部分网盘覆盖，部分网盘是重命名
可以用rclone复制，加个跳过重复文件的参数

**Comment by gujin162** [comment]
> 部分网盘覆盖，部分网盘是重命名
> 可以用rclone复制，加个跳过重复文件的参数

rclone可以秒传吗？

**Reply by pongfcnkl** [reply]
可以
